spec/system/users/edit_spec.rb

```diff
@@ -174,5 +188,12 @@
       expect(ActionMailer::Base.deliveries.first.body.encoded)
         .to match("Your CASA password has been changed.")
     end
+
+    it "displays Casa admin error message if no communication preference is selected" do
```

Comment:
nit: you can just say "admin" here rather than "casa admin"

Reply:
Resolved in c25daba0541612d5fee8d66fa1b57ca3451c4351